diff --git a/node_modules/generouted/src/react-router.tsx b/node_modules/generouted/src/react-router.tsx
index 2418a46..f3a14d0 100644
--- a/node_modules/generouted/src/react-router.tsx
+++ b/node_modules/generouted/src/react-router.tsx
@@ -1,6 +1,15 @@
 import { Fragment, JSX, Suspense } from 'react'
-import { createBrowserRouter, Outlet, RouterProvider, useLocation } from 'react-router'
-import type { ActionFunction, RouteObject, LoaderFunction } from 'react-router'
+import {
+  createBrowserRouter,
+  Outlet,
+  RouterProvider,
+  useLocation,
+} from 'react-router-dom'
+import type {
+  ActionFunction,
+  RouteObject,
+  LoaderFunction,
+} from 'react-router-dom'
 
 import { generateModalRoutes, generatePreservedRoutes, generateRegularRoutes } from './core'
 
